### PR TITLE
added existing module and updated the README.md so no internal stuff …

### PR DESCRIPTION
**Pull request by mattagohni**
…is published.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,147 @@
+provider "aws" {
+  alias = "module"
+  region = "${var.provider_region}"
+  profile = "${var.profile}"
+  assume_role {
+    role_arn = "${var.cross_dev_ops_role_arn}"
```
Comment (sd-lueckel) [inline main.tf]: I think 'assume_role_arn' is a better variable name

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  default = "the AWS profile to use"
+}
+
+variable "vpc_id" {
+  description = "ID of the VPC the security_group should be connected to"
+}
+
+variable "description" {
+  description = "The description of the security group"
+  default = "Basic http Security-Group"
```
Comment (sd-lueckel) [inline variables.tf]: This default value seems to be a little out of context

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  description = "The name of the security group"
+}
+
+variable "custom_tags" {
+  type = "map"
+  default = {}
+}
+
+# Variables for IPv4 cidr_based rules
+variable "cidr_ingress_rules" {
```
Comment (sd-lueckel) [inline variables.tf]: I think we should 'prefix' this variables with 'ipv4' as well

**Review by sd-lueckel — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (mattagohni) [reply main.tf]: changed it to your suggestion.

File: variables.tf (hunk 2 above)
Comment (mattagohni) [reply variables.tf]: I removed it entierly

File: variables.tf (hunk 3 above)
Comment (mattagohni) [reply variables.tf]: i changed it according to the `ipv6`-ones

**Review by sd-lueckel — APPROVED** [review]

**Review by Jnoack331 — APPROVED** [review]

**Pull request merged**